### DISCUSSION TITLE
Saving images and tess files

### DISCUSSION THREAD

**Discussion by asmaaharfoush** [neperfepx/neper#252] [Q&A] [answered]
Hello, 

I wrote a code that produces different morphos with different aspect ratios and orientations, I would like to save the produced images and the tess. files in different folders based on the morpho feature, does neper has a command can to that?

Thanks!

**Comment by dikshanshs** [comment, marked as answer]
@asmaaharfoush I don't know whether the feature exists or not, but you can try building .sh file.

**Reply by rquey** [reply]
The simplest approach is indeed to write a small bash script for this.
